lib/pages/home/home_page.dart

```diff
@@ -15,22 +17,99 @@ class HomePage extends StatefulWidget {
 
 class _HomePageState extends State<HomePage>
     with AutomaticKeepAliveClientMixin {
-  // TODO Update UI later
   @override
   Widget build(BuildContext context) {
     super.build(context);
-    return Center(
-      child: TextButton(
-        style: TextButton.styleFrom(
-          backgroundColor: Colors.blue,
-          padding: const EdgeInsets.all(8),
+    return Material(
+      child: Container(
+        padding: EdgeInsets.only(top: 28.h, bottom: 16.h),
+        color: AppColors.black,
+        child: ListView(
+          children: [
```

Comment:
separate to functions like: _topPage, _latestMovies, _latestSeries... to easy track the page's widgets part.